data/add_media_text_tracks.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE IF NOT EXISTS `media_text_tracks` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `media_id` int(10) unsigned NOT NULL,
+  `datecreated` timestamp NOT NULL,
+  `source` ENUM('amara', 'rev') NULL,
+  `revision_comment` MEDIUMTEXT NULL,
+  PRIMARY KEY (`id`),
+  FOREIGN KEY (`media_id`) REFERENCES media(id),
+  INDEX `media_text_tracks_datecreated` (`datecreated`)
+) ENGINE=MyISAM  DEFAULT CHARSET=latin1;
```

Comment:
The MyISAM engine should not be used on things that require foreign key support.